nodejs/eks/cluster.ts

```diff
@@ -902,6 +903,20 @@ export interface ClusterOptions {
      * The tags to apply to the EKS cluster.
      */
     clusterTags?: InputTags;
+
+    /**
+     * Indicates whether or not to disable auto-naming on the cluster's
+     * physical name. The default is `false` in order to use auto-naming for
+     * the cluster's name, resulting in a physical name with the format
+     * `${name}-eksCluster-0123abcd`.
+     *
+     * Setting this arg to `true` results in a physical name with the format
+     * `${name}` only.
+     *
+     * See for more details:
+     * https://www.pulumi.com/docs/intro/concepts/programming-model/#autonaming
+     */
```

Comment:
Why not just allow the user to optionally pass in a cluster name?

Reply:
Done